scripts/build.sh

```diff
@@ -14,21 +14,12 @@ mkdir -p "$DIR"
 # Transpile CommonJS versions of files
 babel --env-name commonjs src --source-root src --out-dir "$DIR" --copy-files --quiet
 
-# Transpile ESM versions of files for the browser
-babel --env-name esm src --source-root src --out-dir "$DIR/esm-browser" --copy-files --quiet
+# Transpile ESM versions of files
```

Comment:
`Transpile ESM versions of files for use with module bundlers` as we do not yet support native Node.js esmodules.

Reply:
We should support bundlers with node target.

Reply:
The point that I wanted to make is that this is still designated for module bundlers and not https://nodejs.org/api/esm.html

Reply:
So: I agree we should not restrict this to browser but the esm build here is still targeted at module bundlers!

Reply:
Yes, though commonjs browser stuff can be used by browserify unless it supports modules too.